__tests__/app.test.js

```diff
@@ -3,6 +3,7 @@ const request = require("supertest");
 const db = require("../db/connection.js");
 const seed = require("../db/seeds/seed.js");
 const data = require("../db/data/index.js");
+const { get } = require("../app.js");
```

Comment:
I think this might be an unhelpful auto-import as you shouldn't need to import get from app.js.